bin/idamToken.sh

```diff
@@ -1,3 +1,3 @@
 curl -H "Content-type: application/json" -d '{"email":"a@b.com", "password":"password", "forename":"x","surname":"x",  "roles":[{"code":"caseworker","displayName":"caseworker"}]}"' http://betadevaccidamapplb.reform.hmcts.net/testing-support/accounts --silent > /dev/null
-userId=$(curl http://betadevaccidamapplb.reform.hmcts.net/testing-support/accounts/a@b.com --silent | jq -r '.id')
-curl -F id=$userId -F role=caseworker http://betadevaccidamapplb.reform.hmcts.net/testing-support/lease --silent
+userId=$(curl http://localhost:4501/testing-support/accounts/a@b.com --silent | jq -r '.id')
```

Comment:
unfortunately this does not seem to work with the local docker image, you need to use the script above to do it

Reply:
deleted